WcaOnRails/app/views/persons/_results_by_event.html.erb

```diff
@@ -43,11 +43,17 @@
                   </td>
                   <td class="round"><%= t "rounds.#{result.roundTypeId}.cellName" %></td>
                   <td class="place"><%= result.pos %></td>
-                  <td class="single <%= 'pb' if pb_markers[result.id][:single] %>">
+
+                  <% pb_type = {} %>
```

Comment:
Given that this `pb_type` hash is only used twice --one time to fill it and another time to retrieve the value-- maybe it would be better to get ride of this line and the two below and directly use the `pb_type_class_for_result` helper inside the `td class`?